app/population/middleware/set-population.js

```diff
@@ -13,6 +13,7 @@ async function setPopulation(req, res, next) {
     const dailyFreeSpace =
       dailyFreeSpaceByCategory?.[Object.keys(dailyFreeSpaceByCategory)]
 
+    req.categoryName = dailyFreeSpace?.[0]?.category?.title
```

Comment:
Should there be an associated test update for this change?

Reply:
This has now been removed (see organisational switcher comment above)